### ISSUE TITLE
Dota Plus Live Watch

### ISSUE THREAD

**Issue by ShuYukihara** [ValveSoftware/Dota2-Gameplay#10603]
### Description

Tried to watch 5-man party full of my friend but not even 1 of them had the option to watch live, and sometimes only like 2 out of 5 person where the option to watch live is available but not the rest. Tried restarting both DoTA and Steam client but the watch live option is still not available. Please fix, because I bought DotaPlus mostly only for the watch live feature. Thanks.

**Comment by parham2013** [comment]
many people have this issue same as me, it's been an issue for quite a while

**Comment by velieko** [comment]
yeah, it's been a long time now, sometimes no watch live button, sometimes no coach button, sometimes both not even showing, monthly subscription but infuriating bugs.

**Comment by vycarius** [comment]
it's been a long time bug, no watch live button, no coach  button, even better sometimes both not showing, monthly subscription with infuriating bugs..

**Comment by parham2013** [comment]
this issue still persists...
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/74326920/c8e5fb89-4d4b-4f46-90a5-0707ba5279fc)
